model/src/main/java/io/spine/validation/ValidatePolicy.java

```diff
@@ -57,37 +54,47 @@
  *
  * <p>If the message field is invalid, the containing message is invalid as well.
  */
-final class ValidatePolicy extends Policy<FieldOptionDiscovered> {
+final class ValidatePolicy extends Policy<FieldExited> {
 
     @Override
     @React
-    public EitherOf2<SimpleRuleAdded, Nothing> whenever(@External FieldOptionDiscovered event) {
-        Option option = event.getOption();
-        if (!is(option, validate) || !unpack(option.getValue(), BoolValue.class).getValue()) {
-            return EitherOf2.withB(nothing());
+    public EitherOf2<SimpleRuleAdded, Nothing> whenever(@External FieldExited event) {
+        FieldId id = FieldId
+                .newBuilder()
```

Comment:
Please have `.newBuilder() ` on the previous line. IDEA already can align things nicely, and we have options set for this.